frontend/pages/hosts/ManageHostsPage/components/LabelFilterSelect/_styles.scss

```diff
@@ -158,7 +158,7 @@
     min-width: 220px;
 
     &__single-value {
-      max-width: 135px !important; // Must override default styling of .css-qc6sy-singleValue
+      max-width: 210px !important; // Must override default styling of .css-qc6sy-singleValue
```

Comment:
Screenshot of fix:
<img width="321" alt="Screenshot 2024-09-23 at 12 21 06 PM" src="https://github.com/user-attachments/assets/47cbbcb3-99c7-40db-b641-e6ce913ed0a0">